packages/pycryptodome/meta.yaml

```
@@ -0,0 +1,14 @@
+package:
+  name: pycryptodome
+  version: 3.15.0
+source:
+  url: https://files.pythonhosted.org/packages/11/e4/a8e8056a59c39f8c9ddd11d3bc3e1a67493abe746df727e531f66ecede9e/pycryptodome-3.15.0.tar.gz
+  sha256: 9135dddad504592bcc18b0d2d95ce86c3a5ea87ec6447ef25cfedea12d6018b8
+test:
```

I think the following should fix it:
```suggestion
build:
  exports: "requested"
test:
```